colors/build.js

```diff
@@ -2,29 +2,43 @@ const fs = require('fs');
 const colors = require('./photon.json');
 const metadata = require('./package.json');
 
-function createColor(color, element) {
-  var rv = [];
-  for (var variant in element) {
+function createColor(color, element, prefix) {
+  const rv = [];
+
+  for (const variant in element) {
     if (element.hasOwnProperty(variant)) {
-      var value = element[variant];
-      rv.push(`  --${color}-${variant}: ${value};\n`);
+      const value = element[variant];
+      rv.push(`${prefix}${color}-${variant}: ${value};\n`);
     }
   }
+  rv.push('\n');
   return rv;
 }
 
-let rv = [`/* Firefox Colors CSS Variables v${metadata.version} */
+
+const cssOutput = [`/* Firefox Colors CSS Variables v${metadata.version} */
 
 :root {
 `];
-for (var color in colors) {
+
+const sassOutput = [`/* Firefox Colors SCSS Variables v${metadata.version} */
```

Comment:
Perhaps we should add a comment here letting people know that these shouldn't be modified, as per the other discussion?